recipe/meta.yaml

```diff
@@ -38,16 +37,14 @@ requirements:
     - natsort
     - lz4
     - typing_extensions
-    - python-isal
+    - python-isal [x86_64]
```

Comment:
`osx-arm64` is also supported, see https://github.com/danielhrisca/asammdf/commit/78aac196b16989f6324184ef3f37e9f34cdca500

Reply:
Could you also group required and optional dependencies (for readability) and add a comment on why optional are included (encoding, speedup and so on) from https://github.com/danielhrisca/asammdf/blob/c93de510f8e06526152feefa6d54575b3d29722f/doc/intro.rst#dependencies?

Reply:
Could you fix `python-isal   # [x86_64]` because of missing `#`?